core/new-gui/src/app/workspace/component/result-panel/console-frame/console-frame.component.ts

```diff
@@ -84,6 +94,8 @@ export class ConsoleFrameComponent implements OnInit, OnChanges {
     const breakpointTriggerInfo = this.executeWorkflowService.getBreakpointTriggerInfo();
 
     if (this.operatorId) {
+      this.workerIds = this.executeWorkflowService.getWorkerIds(this.operatorId);
```

Comment:
can we make sure that when the worker id is first assigned, the console is always re-rendered?

Reply:
the logic here is every time the console is re-rendered, it request for the latest workerId information from executeWorkflowService. I think it is fine?